vcd/resource_vcd_vapp_org_network_test.go

```diff
@@ -147,7 +147,7 @@ resource "vcd_network_routed" "{{.NetworkName}}" {
 resource "vcd_vapp_org_network" "{{.resourceName}}" {
   org                = "{{.Org}}"
   vdc                = "{{.Vdc}}"
-  vapp_name          = "{{.vappName}}"
+  vapp_name          = vcd_vapp.{{.vappName}}.name
```

Comment:
Was this a bug? How come we didn't hit it while testing?

Reply:
depends_on was handling it